src/html/partials/story/intro.hbs

```diff
@@ -55,16 +56,16 @@
 </p>
 </div> <!-- /turn-back-time -->
 
-<hr/>
 <h3>1959</h3>
 
 <div class="step" data-song="The Happy Organ"> <!-- instrumental -->
+	<div class="step">
+	<p>
+		Sometimes an extreme value of a single attribute can be enough to make a song notably atypical, as with “The Happy Organ,” which topped the charts in 1959.
+	</p>
+	<div class="player" data-song="The Happy Organ"></div>
+</div>
```

Comment:
This seems to have broken the 'cardiness' of the text. I'll look into fixing it. I think I need to think more carefully about the semantics of div.step, p.step, div.card, elements nested within them, and how they participate in the scrolling scenes.

Reply:
Okay, just pushed a change that fixed this (and also fixed some of the wonky card timing). This approach (of enclosing the text and the player div in another div) should generally work for compactifying the arrangement of the players.